### DISCUSSION TITLE
Napalm for Extreme EXOS

### DISCUSSION THREAD

**Discussion by johnbrannstrom2** [napalm-automation/napalm#2019] [Polls]
Building on [napalm-exos](https://github.com/iaa-inc/napalm-exos), I have participated in building an implementation of napalm complete with tests for Extreme EXOS. Is there any interest of adding this to the project as a community driver?

**Comment by ktbyers** [comment]
Standard process would be to create a NAPALM community driver for it:

https://github.com/orgs/napalm-automation-community/repositories?type=all

But yeah we wouldn't include it in core NAPALM.

**Reply by johnbrannstrom2** [reply]
I realized that after, writing the poll. I have adjusted the poll accordingly. Adding it as a community driver would be fit for purpose. I will get back as soon as a decision has been made regarding the status open source status of the EXOS driver.
